packages/flutter/lib/src/material/dropdown_menu.dart

```diff
@@ -367,6 +383,22 @@ class DropdownMenu<T> extends StatefulWidget {
 
   final Object? _inputDecorationTheme;
 
+  /// The builder function used to create the [InputDecoration] passed to the text field.
+  ///
+  /// If provided and the resulting [InputDecoration.suffixIcon] is null,
+  /// the [InputDecoration.suffixIcon] is set to an [IconButton] which uses [trailingIcon]
+  /// and [selectedTrailingIcon] if defined, or [Icons.arrow_drop_down] and
+  /// [Icons.arrow_drop_up] otherwise.
```

Comment:
nit: Maybe this.
```suggestion
/// If a value is provided for this property and the resulting [InputDecoration.suffixIcon] is null, a default [IconButton] is assigned as the suffix icon. This button's icon will use [trailingIcon] and [selectedTrailingIcon] if those are explicitly defined; otherwise, it defaults to [Icons.arrow_drop_down] for the collapsed state and [Icons.arrow_drop_up] for the expanded state.
```